client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -70,6 +72,8 @@ export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) =>
 
 		const allEventProps = {
 			...trackingEventsProps,
+			from: fromUrl,
```

Comment:
Nit: did you consider not including the prop in case the `fromUrl` variable is empty?
```suggestion
			...( fromUrl !== '' ? { from: fromUrl } : {} )
```